README.md

```diff
@@ -66,6 +67,15 @@ The following provisioning mechanisms are currently supported:
 Which kind of cluster is chosen depends on the requirements of your test.
 For example, using containers allows to easily test against different broker versions. 
 
+---
+**NOTE**
+
+In case you use podman for testcontainers, some tips must be taken into account:
+* `TESTCONTAINERS_RYUK_DISABLED=true` env variable shall be declared as per https://github.com/containers/podman/issues/7927#issuecomment-731525556
+* `podman-plugins` linux package shall be installed on your machine in order to communicate containers among each other.
```

Comment:
Can we enforce this as part of the build? I'm happy for that to be in a separate PR.

Reply:
The env variable had been included by @k-wall here: https://github.com/kroxylicious/kroxylicious-junit5-extension/blob/main/pom.xml#L260